app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx

```diff
@@ -67,7 +67,7 @@ const messages = defineMessages({
   disablePluginMessage: {
     id: 'PluginItem.disablePluginMessage',
     defaultMessage:
-      'Are you sure you want to disable a plugin {pluginName}? If you disable plugin, information about it will be hidden on the Project Settings and users can not interact with it',
+      'Are you sure you want to disable a plugin {pluginName}? If you disable plugin, information about it will be hidden on the {pluginLocation} and users can not interact with it',
```

Comment:
Please run `manage:translations` to update corresponding messages.

Reply:
I ran it, but nothing was changed

Reply:
Seems like you need to update it manually to support new parameter.